src/lib/vm_types.rs

```diff
@@ -155,24 +158,27 @@ pub struct Step {
 // interruptions are events that prevent steppping from progressing.
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
 pub enum Interruption {
+    Done(Value),
```

Comment:
I just realized that we have a `Done` interruption and a `Done` signal. Maybe we could rename `Signal::Done` to something like `Signal::Continue`?

Reply:
They mean the same thing, so it's sensible to me that they are named the same.

The `Done` in Signal is like a promoted `Done` interruption, to avoid having to do a deep pattern match as the consumer. 

I don't anticipate other cases to be treated like this, just `Done`.